utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -297,7 +303,7 @@ def generate_baseline(self):
         basegen_dir = os.path.join(baselineroot, self._case.get_value("BASEGEN_CASE"))
         for bdir in (baselineroot, basegen_dir):
             if not os.path.isdir(bdir):
-                append_status("GFAIL %s baseline\n" % self._case.get_value("CASEBASEID"),
+                append_status("FAIL %s generate\n" % self._case.get_value("CASEBASEID"),
```

Comment:
When generating a baseline fails, we have typically called this a "GFAIL", but this is basically the same thing as failing the "generate" phase, so that's what I marked this as. The most likely reason to fail at this point would, I think, be if there's some kind of permissions issue that stops baseline generation, or if there's a scripts bug. To be honest, though, I have not thought about this too deeply, since during the testing I was doing I was only comparing rather than generating baselines. Getting rid of the "GFAIL" was an afterthought.

In contrast, we probably do want "BFAIL" to be a separate status because it doesn't necessarily signify an error. For instance, if you've just added a new test and there is no baseline for that configuration, a "BFAIL" is perfectly normal. I'm wondering if I should add it to `reduce_stati` to somehow emphasize the fact that (like "NLFAIL") it's not as severe as a "FAIL" or "DIFF" status.